### PR TITLE
Update Project Loved Wiki Page

### PR DESCRIPTION
**Pull request by huuishuu**
Add 30s drain time requirement to the eligibility criteria.

### CODE REVIEW

File: wiki/Project_Loved/en.md [hunk 1]
```diff
@@ -28,6 +28,7 @@ There are some very minimal criteria that need to be met for beatmaps to be nomi
 - Its creator is neither banned nor restricted
 - At least one difficulty can be passed
 - It has at least 30 favourites (only applies to osu!standard maps)
+- Every difficulty of a beatmapset must have a minimum [drain time](/wiki/en/Gameplay/Drain_time) of 30 seconds.
```
Comment (MegaApplePi) [inline wiki/Project_Loved/en.md:31]:
```suggestion
- Every difficulty of a beatmapset must have a minimum [drain time](/wiki/Gameplay/Drain_time) of 30 seconds.
```
The link should not have the `en/` part. Also, are we also moving away from using beatmapset?

Comment (huuishuu) [reply wiki/Project_Loved/en.md]: I copied and pasted this exact point from the [Ranking Criteria](https://osu.ppy.sh/wiki/en/Ranking_Criteria) page so I'm not entirely sure.

File: wiki/Project_Loved/en.md [hunk 2]
```diff
@@ -28,6 +28,7 @@ There are some very minimal criteria that need to be met for beatmaps to be nomi
 - Its creator is neither banned nor restricted
 - At least one difficulty can be passed
 - It has at least 30 favourites (only applies to osu!standard maps)
+- Every difficulty of a beatmapset must have a minimum [drain time](/wiki/Gameplay/Drain_time) of 30 seconds.
```
Comment (MegaApplePi) [inline wiki/Project_Loved/en.md:31]: Oh, I did not realise that. I think it should be phrased a bit differently to match the above points. Maybe something like this?
```suggestion
- Every difficulty has at least 30 seconds of [drain time](/wiki/Gameplay/Drain_time)
```
If you don't think so, then I'll go ahead and merge.

Comment (huuishuu) [reply wiki/Project_Loved/en.md]: Sure, it looks more uniform that way.

**Review by MegaApplePi — APPROVED** [review]

File: wiki/Project_Loved/en.md (hunk 1 above)
Comment (TicClick) [reply wiki/Project_Loved/en.md]: > are we also moving away from using beatmapset?

we totally are, but I need to organize the existing `Beatmap`/`Beatmapset` articles first, then play find-and-replace

**Pull request merged**